src/hooks/permanentFilter.ts

```diff
@@ -2,4 +2,21 @@ import { useContext } from 'react';
 
 import { PermanentFilterContext } from '../PermanentFilterProvider';
 
-export const usePermanentFilter = () => useContext(PermanentFilterContext);
+export const usePermanentFilter = () => {
+  const {
+    dataProviderState,
+    dataProviderDispatch,
+    mowasRegionalKeysState,
+    mowasRegionalKeysDispatch
+  } = useContext(PermanentFilterContext);
+
+  const excludeDataProviderIds = dataProviderState;
+  const excludeMowasRegionalKeys = mowasRegionalKeysState;
```

Comment:
why are you renaming these?

Reply:
Because according to the ticket, we have to use it as follows. For this reason, I have decided to change the name.

```
  const { excludeDataProviderIds, excludeMowasRegionalKeys } = usePermanentFilter();
```

Reply:
hm ok, but then we do not need to create to variables. you can rename in the return directly, right?!

Reply:
Yes, that's right. We don't need an extra variable for this. 👍